src/components/Tabs.tsx

```diff
@@ -102,7 +99,7 @@ export default class Tabs extends React.Component<TabsProps, TabsState> {
   render(): JSX.Element {
     let [navs, content] = this.makeTabs();
     return (
-      <section className="tabs">
+      <section className="reusable-tabs">
```

Comment:
I think `tabs` is fine. If you use `reusable-` here then it'd be good to use the same namespace for the other components to have a consistent naming convention.

Reply:
The point was to be able to go ahead and use the styling on Storybook even though we haven't yet implemented this component in the interfaces; the interfaces already have things with className `tabs`, which we don't want this to overwrite.  (We had commented out the import statement for `tabs.scss` in `app.scss` in order to prevent this...but that's why the styling wasn't being used in Storybook.)  In other words, changing the class names is just a quick fix so that we can send the Storybook link to the design system forum ASAP; afterwards, we can reassess what kind of naming convention we want.

Reply:
Right, and I think those `tab` classes come from bootstrap. Next time let me know that's the goal for the PR or mention that it's temporary.

Reply:
Full explanation is in the linked JIRA ticket.